src/overlays/actors/ovl_Bg_Mizu_Movebg/z_bg_mizu_movebg.h

```diff
@@ -4,6 +4,18 @@
 #include "ultra64.h"
 #include "global.h"
 
+typedef enum BgMizuMovebgType {
+    /* 0 */ MIZUMOVEBG_TYPE_FLOATING_PLATFORM_OUTSIDE_CENTER_PILLAR,
+    /* 1 */ MIZUMOVEBG_TYPE_FLOATING_PLATFORM_WEST,
+    /* 2 */ MIZUMOVEBG_TYPE_FLOATING_PLATFORM_INSIDE_CENTER_PILLAR,
+    /* 3 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_BOSS_ROOM, // Dragon Statue leading to the Water Temple boss room
+    /* 4 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_2,         // Unused on Master Quest, same behavior as ``MIZUMOVEBG_TYPE_DRAGON_STATUE_3``
+    /* 5 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_3,         // Dragon Statues used in Water Temple puzzles
+    /* 6 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_4,         // Unused on Master Quest, same behavior as ``MIZUMOVEBG_TYPE_DRAGON_STATUE_3``
```

Comment:
personally dont feel its worth mentioning "unused in master quest" in the comments

Reply:
when I wrote the comments I was thinking I should mention it's unused but then I thought maybe it's used on vanilla for some reasons

I don't have a strong opinion on this so I can either remove the whole thing or just keep "unused" or keep it like it is currently

Reply:
Id say remove any mention of unused for now. If it is unused even after we work on non-mq, then something can be said